<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logImport</class>
 <widget class="QDialog" name="logImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Log Import</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fileSelectLabel">
     <property name="text">
      <string>Select a log file to import.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fileButton">
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="joinCheck">
     <property name="text">
      <string>Is joining log file</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="importButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Warning: Importing will clear the current recorded data.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
